libretroshare/src/gxs/rsgds.h

```diff
@@ -48,13 +48,12 @@ class RsGxsSearchModule  {
  * This allows modification of local
  * meta data items of a message
  */
-class MsgLocMetaData {
+struct MsgLocMetaData {
+	MsgLocMetaData() {}

```

Comment:
What I meant  you should remove this empty constructor, and probably the compiler will generate it automatically. If not please use `MsgLocMetaData() = default;` that tell explicitly to the compiler to generate it.

Reply:
Done